apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/style.scss

```diff
@@ -0,0 +1,128 @@
+@import '~@wordpress/base-styles/colors';
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+
+$sidebarWidth: 272px;
+$border: 1px solid $light-gray-500;
+$transitionPeriod: 200ms;
```

Comment:
Tiny nit: this and `$sidebarWidth` are using camel case where the Gutenberg variable names use kebab case. Is it worth making them consistent?

Reply:
Yup, I was confused because I was thinking of this more as belonging to the Calypso repo and following Calypso standards. [But I was wrong about them anyway](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#general-syntax-and-writing-rules) 😅